{
    "DWF": {
        "references": [
            "https://www.suse.com/security/cve/CVE-2020-25658/",
            "https://access.redhat.com/security/cve/CVE-2020-25658"
        ]
    },
    "MITRE": {
        "CVE_data_meta": {
            "ASSIGNER": "secalert@redhat.com",
            "ID": "CVE-2020-25658",
            "STATE": "PUBLIC"
        },
        "affects": {
            "vendor": {
                "vendor_data": [
                    {
                        "product": {
                            "product_data": [
                                {
                                    "product_name": "python-rsa",
                                    "version": {
                                        "version_data": [
                                            {
                                                "version_value": "after 3.0 (inclusive)"
                                            }
                                        ]
                                    }
                                }
                            ]
                        },
                        "vendor_name": "Sybren A. St\u00fcvel"
                    }
                ]
            }
        },
        "data_format": "MITRE",
        "data_type": "CVE",
        "data_version": "4.0",
        "description": {
            "description_data": [
                {
                    "lang": "eng",
                    "value": "It was found that python-rsa is vulnerable to Bleichenbacher timing attacks. An attacker can use this flaw via the RSA decryption API to decrypt parts of the cipher text encrypted with RSA."
                }
            ]
        },
        "impact": {
            "cvss": [
                [
                    {
                        "vectorString": "7.5/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
                        "version": "3.0"
                    }
                ]
            ]
        },
        "problemtype": {
            "problemtype_data": [
                {
                    "description": [
                        {
                            "lang": "eng",
                            "value": "CWE-385"
                        }
                    ]
                }
            ]
        },
        "references": {
            "reference_data": [
                {
                    "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25658",
                    "refsource": "CONFIRM",
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25658"
                },
                {
                    "name": "https://github.com/sybrenstuvel/python-rsa/issues/165",
                    "refsource": "MISC",
                    "url": "https://github.com/sybrenstuvel/python-rsa/issues/165"
                },
                {
                    "name": "FEDORA-2021-783a157adc",
                    "refsource": "FEDORA",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/2SAF67KDGSOHLVFTRDOHNEAFDRSSYIWA/"
                },
                {
                    "name": "FEDORA-2021-c1fef03e71",
                    "refsource": "FEDORA",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QY4PJWTYSOV7ZEYZVMYIF6XRU73CY6O7/"
                },
                {
                    "name": "FEDORA-2021-15e50503d6",
                    "refsource": "FEDORA",
                    "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/APF364QJ2IYLPDNVFBOEJ24QP2WLVLJP/"
                }
            ]
        }
    },
    "NVD": {
        "configurations": {
            "CVE_data_version": "4.0",
            "nodes": [
                {
                    "children": [],
                    "cpe_match": [
                        {
                            "cpe23Uri": "cpe:2.3:a:python-rsa_project:python-rsa:*:*:*:*:*:*:*:*",
                            "cpe_name": [],
                            "versionEndExcluding": "4.7",
                            "versionStartIncluding": "2.1",
                            "vulnerable": true
                        }
                    ],
                    "operator": "OR"
                },
                {
                    "children": [],
                    "cpe_match": [
                        {
                            "cpe23Uri": "cpe:2.3:a:redhat:openstack_platform:13.0:*:*:*:*:*:*:*",
                            "cpe_name": [],
                            "vulnerable": true
                        },
                        {
                            "cpe23Uri": "cpe:2.3:a:redhat:openstack_platform:16.0:*:*:*:*:*:*:*",
                            "cpe_name": [],
                            "vulnerable": true
                        }
                    ],
                    "operator": "OR"
                }
            ]
        },
        "cve": {
            "CVE_data_meta": {
                "ASSIGNER": "secalert@redhat.com",
                "ID": "CVE-2020-25658"
            },
            "data_format": "MITRE",
            "data_type": "CVE",
            "data_version": "4.0",
            "description": {
                "description_data": [
                    {
                        "lang": "en",
                        "value": "It was found that python-rsa is vulnerable to Bleichenbacher timing attacks. An attacker can use this flaw via the RSA decryption API to decrypt parts of the cipher text encrypted with RSA."
                    }
                ]
            },
            "problemtype": {
                "problemtype_data": [
                    {
                        "description": [
                            {
                                "lang": "en",
                                "value": "CWE-385"
                            }
                        ]
                    }
                ]
            },
            "references": {
                "reference_data": [
                    {
                        "name": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25658",
                        "refsource": "CONFIRM",
                        "tags": [
                            "Issue Tracking",
                            "Third Party Advisory"
                        ],
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25658"
                    },
                    {
                        "name": "https://github.com/sybrenstuvel/python-rsa/issues/165",
                        "refsource": "MISC",
                        "tags": [
                            "Exploit",
                            "Issue Tracking",
                            "Third Party Advisory"
                        ],
                        "url": "https://github.com/sybrenstuvel/python-rsa/issues/165"
                    },
                    {
                        "name": "FEDORA-2021-783a157adc",
                        "refsource": "FEDORA",
                        "tags": [],
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/2SAF67KDGSOHLVFTRDOHNEAFDRSSYIWA/"
                    },
                    {
                        "name": "FEDORA-2021-c1fef03e71",
                        "refsource": "FEDORA",
                        "tags": [],
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QY4PJWTYSOV7ZEYZVMYIF6XRU73CY6O7/"
                    },
                    {
                        "name": "FEDORA-2021-15e50503d6",
                        "refsource": "FEDORA",
                        "tags": [],
                        "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/APF364QJ2IYLPDNVFBOEJ24QP2WLVLJP/"
                    }
                ]
            }
        },
        "impact": {
            "baseMetricV2": {
                "acInsufInfo": false,
                "cvssV2": {
                    "accessComplexity": "MEDIUM",
                    "accessVector": "NETWORK",
                    "authentication": "NONE",
                    "availabilityImpact": "NONE",
                    "baseScore": 4.3,
                    "confidentialityImpact": "PARTIAL",
                    "integrityImpact": "NONE",
                    "vectorString": "AV:N/AC:M/Au:N/C:P/I:N/A:N",
                    "version": "2.0"
                },
                "exploitabilityScore": 8.6,
                "impactScore": 2.9,
                "obtainAllPrivilege": false,
                "obtainOtherPrivilege": false,
                "obtainUserPrivilege": false,
                "severity": "MEDIUM",
                "userInteractionRequired": false
            },
            "baseMetricV3": {
                "cvssV3": {
                    "attackComplexity": "HIGH",
                    "attackVector": "NETWORK",
                    "availabilityImpact": "NONE",
                    "baseScore": 5.9,
                    "baseSeverity": "MEDIUM",
                    "confidentialityImpact": "HIGH",
                    "integrityImpact": "NONE",
                    "privilegesRequired": "NONE",
                    "scope": "UNCHANGED",
                    "userInteraction": "NONE",
                    "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:N/A:N",
                    "version": "3.1"
                },
                "exploitabilityScore": 2.2,
                "impactScore": 3.6
            }
        },
        "lastModifiedDate": "2021-09-25T01:15Z",
        "publishedDate": "2020-11-12T14:15Z"
    }
}